spacifier.js

```diff
@@ -0,0 +1,8 @@
+const spacify = (str) => {
+	let splitResult = str.split('..');
```

Comment:
If you want to use ,. or ., separator interchangeable

```suggestion
	let splitResult = str.split(/\.,|,\./);
```

Reply:
By the power of democracy we'll keep using .. for now

Reply:
Alright